userguide/storage.rst

```diff
@@ -1718,28 +1719,27 @@ summarizes the available options.
-   |                |                             | :guilabel:`FETCH DATASTORES` to populate the menu, then select the datastore to be synchronized.            |
-   |                |                             |                                                                                                             |
-   +----------------+-----------------------------+-------------------------------------------------------------------------------------------------------------+
+   +----------------+----------------+-------------------------------------------------------------------------------------------------------------+
+   | Setting        | Value          | Description                                                                                                 |
+   +================+================+=============================================================================================================+
+   | Hostname       | string         | Enter the IP address or hostname of the VMware host. When clustering, use the IP of the vCenter server for  |
+   |                |                | the cluster.                                                                                                |
+   +----------------+----------------+-------------------------------------------------------------------------------------------------------------+
+   | Username       | string         | Enter the username on the VMware host with permission to snapshot virtual machines.                         |
+   +----------------+----------------+-------------------------------------------------------------------------------------------------------------+
+   | Password       | string         | Enter the password associated with :guilabel:`Username`.                                                    |
+   +----------------+----------------+-------------------------------------------------------------------------------------------------------------+
+   | ZFS Filesystem | browse button  | :guilabel:`Browse` to the filesystem to snapshot.                                                           |
+   +----------------+----------------+-------------------------------------------------------------------------------------------------------------+
+   | Datastore      | drop-down menu | After entering the :guilabel:`Hostname`, :guilabel:`Username`, and :guilabel:`Password`, click              |
+   |                |                | :guilabel:`FETCH DATASTORES` to populate the menu, then select the datastore to be synchronized.            |
+   +----------------+----------------+-------------------------------------------------------------------------------------------------------------+
+
+
+%brand% attempts to connect to the VMware host after the credentials are
+entered and populate the :guilabel:`ZFS Filesystem` and
+:guilabel:`Datastore` drop-down menus. Choosing a datastore also selects
```

Comment:
I read the first sentence several times before understanding. Suggestion: make 2 sentences-
```
%brand% attempts to connect to the VMware host after the credentials
are entered. It also populates the :guilabel:`ZFS Filesystem` and
:guilabel:`Datastore` drop-down menus.
```
or something like that...

Reply:
Yes, break the two thoughts into two sentences.  Also, "attempts" implies that it often fails, which is probably not true.
`After the credentials are entered, %brand% connects to the VMware host and populates the ...`